iib/web/api_v1.py

```diff
@@ -257,6 +262,39 @@ def get_builds():
         batch_id = Batch.validate_batch(batch_id)
         query = query.filter_by(batch_id=batch_id)
 
+    if request_type:
+        RequestTypeMapping.validate_type(request_type)
+        request_type = request_type.replace('-', '_')
+        request_type_int = RequestTypeMapping.__members__[request_type].value
+        query = query.filter_by(type=request_type_int)
+
+    if user:
+        # join with the user table to filter on username
+        # request table only has the user_id
+        query = query.join(Request.user)
+        query = query.filter(User.username == user)
+
+    if index_image:
+        # Get the image id of the image to be searched
+        image_result = Image.query.filter_by(pull_specification=index_image).first()
+        if image_result:
+            # join with the Request* tables to get the response as image_ids are stored there
+            query = query.outerjoin(
+                RequestCreateEmptyIndex, Request.id == RequestCreateEmptyIndex.id
+            )
+            query = query.outerjoin(RequestAdd, Request.id == RequestAdd.id)
+            query = query.outerjoin(RequestMergeIndexImage, Request.id == RequestMergeIndexImage.id)
```

Comment:
CAn you rename it from query to something else?

Reply:
I think it has to be named query to keep it consistent and as Tulsi mentioned, we are using the same variable name in the function. Let's keep it as is.